gcloud-java-contrib/gcloud-java-nio/README.md

```diff
@@ -0,0 +1,174 @@
+NIO Filesystem Provider for Google Cloud Storage
+================================================
+
+Java idiomatic NIO client for [Google Cloud Storage](https://cloud.google.com/storage/).
+
+This client library allows you to easily interact with Google Cloud Storage using Java's
+standard file system API.
+
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/gcloud-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/gcloud-java)
+[![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/gcloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/gcloud-java?branch=master)
+[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/gcloud-java-nio.svg)]( https://img.shields.io/maven-central/v/com.google.cloud/gcloud-java-nio.svg)
```

Comment:
The snipped you gave me had one Codacy and two Dependency badges. I put the Codacy badge, but only one Dependency badge.